rosbag2_tests/test/rosbag2_tests/record_fixture.hpp

```diff
@@ -87,6 +87,24 @@ class RecordFixture : public TemporaryDirectoryFixture
     return root_bag_path_ / (get_bag_file_name(split_index) + ".db3");
   }
 
+  void wait_for_metadata(int timeout_in_sec = 5)
```

Comment:
It seems that `timeout_in_sec` is never used, so we don't need this parameter, we can just set `5` as a constant.
https://martinfowler.com/bliki/Yagni.html

Reply:
Would it make sense to define it as a constant somewhere or just use `5` directly?